src/04-modality-specific-files/11-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,402 @@
+# Functional Near-Infrared Spectroscopy
+
+Support for functional Near-Infrared Spectroscopy (fNIRS) was developed as a
+[BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).
+Please see [Citing BIDS](../01-introduction.md#citing-bids)
+on how to appropriately credit this extension when referring to it in the
+context of the academic literature.
+
+Several [example fNIRS datasets](https://github.com/bids-standard/bids-examples#fnirs-datasets)
+have been formatted using this specification and can be used for practical guidance when curating a new dataset.
+
+## fNIRS recording data
+
+{{ MACROS___make_filename_template(datatypes=["nirs"], suffixes=["nirs", "events", "channels", "optodes", "coordsystem"]) }}
+
+Only the *Shared Near Infrared Spectroscopy Format* ([SNIRF](https://github.com/fNIRS/snirf))
+file specification is supported in BIDS. The SNIRF
```

Comment:
My understanding was that all BEPs once merged were BIDS and that there was no fNIRS-BIDS. I am happy to use which ever name is most consistent with other aspects of BIDS.

Reply:
> My understanding was that all BEPs once merged were BIDS and that there was no fNIRS-BIDS

That's correct ... we sometimes still talk about EEG-BIDS or MEG-BIDS e.g., in the publications or informally. As this BEP was called NIRS, I'd vote to change this to NIRS-BIDS, rather than with the "f" prefix.